### PR TITLE
Fix eslint warnings

### LINKED ISSUE #128: Fix all webpack warnings
[TODO: Description]

### PR DESCRIPTION
**Pull request by sunify**
Fixes #128

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,34 +12,29 @@ import SendByScan from './components/SendByScan';
...
 import burnerlogo from './burnerwallet.png';
-import BurnWallet from './components/BurnWallet'
-import Exchange from './components/Exchange'
+import BurnWallet from './components/BurnWallet';
+import Exchange from './components/Exchange';
 import Bottom from './components/Bottom';
-import customRPCHint from './customRPCHint.png';
-import incogDetect from './services/incogDetect.js'
+import incogDetect from './services/incogDetect.js';
 import { Card, Box, ThemeProvider } from 'rimble-ui';
```
Comment (iamonuwa) [inline src/App.js]: Can we have npm packages first before custom components and services?

Comment (sunify) [reply src/App.js]: Sure. Create an issue :-)

It's out of this PR and original issue scopes

File: src/components/Header.js [hunk 2]
```diff
@@ -1,128 +1,158 @@
...
+          src={burnerloader}
+          style={{ maxHeight: 50, opacity: 0.25, marginLeft: -20 }}
+          alt=""
+        />
       </div>
-    )
-  }else{
+    );
+  } else {
     /*moneyDisplay = (
```
Comment (iamonuwa) [inline src/components/Header.js]: Can we remove this?

File: src/components/Receipt.js [hunk 3]
```diff
@@ -1,107 +1,180 @@
...
+    if (this.props.receipt && this.props.receipt.daiposOrderId) {
+      console.log('This was a daipos Order... ping their server for them...');
       // https://us-central1-daipos.cloudfunctions.net/transactionBuffer?orderId=0JFmycULnk9kAboK5ESg&txHash=0x8c831cd5cbc8786982817e43a0a77627ad0b12eaa92feff97fb3b7e91c263b1c&networkId=100
-      let url = "https://us-central1-daipos.cloudfunctions.net/transactionBuffer?orderId="+this.props.receipt.daiposOrderId+"&txHash="+this.props.receipt.result.transactionHash+"&networkId=100"
-      console.log("url:",url)
-      fetch(url).then(r => r.json()).then((response)=>{
-         console.log("Finished hitting the Ching servers:",response)
-       })
+      let url =
+        'https://us-central1-daipos.cloudfunctions.net/transactionBuffer?orderId=' +
```
Comment (iamonuwa) [inline src/components/Receipt.js]: Would be nice to use string literals than concatenating operators.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: src/components/RecentTransactions.js [hunk 4]
```diff
@@ -1,181 +1,225 @@
...
-          <span style={{fontSize:14}}>
-            {message}
-          </span>
-        )
+        toBlockie = <span style={{ fontSize: 14 }}>{message}</span>;
       }
 
-      if(count++<max){
+      if (count++ < max) {
         //if(txns.length>0){
```
Comment (iamonuwa) [inline src/components/RecentTransactions.js]: Please remove if not in use

File: src/components/RequestFunds.js [hunk 5]
```diff
@@ -1,88 +1,124 @@
 import React from 'react';
-import Ruler from "./Ruler";
-import Balance from "./Balance";
-import Blockies from 'react-blockies';
-import {CopyToClipboard} from "react-copy-to-clipboard";
+import { CopyToClipboard } from 'react-copy-to-clipboard';
 import i18n from '../i18n';
 import RecentTransactions from './RecentTransactions';
 // const QRCode = require('qrcode.react');
```
Comment (iamonuwa) [inline src/components/RequestFunds.js]: Remove if not in use

File: src/components/RequestFunds.js [hunk 6]
```diff
@@ -1,88 +1,124 @@
...
+                borderRadius={1}
+              >
+                <QRCode value={qrValue} size={'100%'} renderAs={'svg'} />
+              </Flex>
+              <Box mt={3}>
+                <Input type="url" readOnly value={qrValue} width={1} />
+              </Box>
 
-            {/* <div className="input-group">
+              {/* <div className="input-group">
```
Comment (iamonuwa) [inline src/components/RequestFunds.js]: please remove if not in use

File: src/components/WithdrawFromPrivate.js [hunk 7]
```diff
@@ -1,214 +1,248 @@
...
+    let { fromAddress, amount, metaAccount } = this.state;
+    const { tokenSendV2, address, web3, xdaiweb3, pDaiTokenAddr } = this.props;
+
+    if (this.state.canWithdraw) {
+      console.log('SWITCH TO LOADER VIEW...');
+      this.props.changeView('loader');
+      setTimeout(() => {
+        window.scrollTo(0, 0);
+      }, 60);
+      //console.log("metaAccount",this.state.metaAccount,"amount",this.props.web3.utils.toWei(amount,'ether'))
```
Comment (iamonuwa) [inline src/components/WithdrawFromPrivate.js]: Please remove.

**Comment by TimDaub** [comment]
I'm assuming you ran Prettier on the code base. While I'm hugely pro running Prettier, I think doing this change might lead to problems later on.

The following scenarios will cause problems IMO:

- We want to update Leap's burner-wallet with upstream;
- We want to contribute to upstream

In both cases, this PR will create huge merge conflict diffs.

What I meant by creating #128 (sorry for not filling out the issues description correctly :() is that we fix the about 20 eslint errors manually for now without changing much the code formatting.

Thoughts?

**Comment by sunify** [comment]
@TimDaub i think you're right :-) I'll update the pr without prettier formatting

Also, i'll send PR to upstream with these fixes, prettier formatting and pre-commit hook to maintain it

File: src/App.js [hunk 8]
```diff
@@ -1714,6 +1700,8 @@ let sortByBlockNumber = (a,b)=>{
   return 0
 }
 
+// ToDo: do not mutate native prototypes
```
Comment (TimDaub) [inline src/App.js]: We could simply file an issue and label it as bug for now. What do you think?

File: src/components/Advanced.js [hunk 9]
```diff
@@ -20,10 +19,10 @@ export default class Advanced extends React.Component {
     }
   }
   render(){
-    let {isVendor, balance, address, privateKey, changeAlert, changeView, goBack, setPossibleNewPrivateKey} = this.props
+    let {isVendor, balance, privateKey, changeAlert, changeView, setPossibleNewPrivateKey} = this.props
 
     let url = window.location.protocol+"//"+window.location.hostname
-    if(window.location.port&&window.location.port!=80&&window.location.port!=443){
+    if(window.location.port&&window.location.port!==80&&window.location.port!==443){
```
Comment (TimDaub) [inline src/components/Advanced.js]: TIL: `window.location.port` exists 😂

File: src/components/Balance.js [hunk 10]
```diff
@@ -19,19 +17,6 @@ export  default ({icon, text, selected, amount, address, dollarDisplay}) => {
     opacity=0.05
   }
 
-  let iconDisplay
```
Comment (TimDaub) [inline src/components/Balance.js]: Why did you remove this whole section?

File: src/components/Exchange.js [hunk 11]
```diff
@@ -240,23 +228,6 @@ export default class Exchange extends React.Component {
         }
       }
     }
-    const xDaiToEthGasStationEnabled = false;
```
Comment (TimDaub) [inline src/components/Exchange.js]: 👍

File: src/components/Exchange.js [hunk 12]
```diff
@@ -747,19 +718,10 @@ export default class Exchange extends React.Component {
         </a>
       </span>
     )
-    let xdaiCancelButton = (
```
Comment (TimDaub) [inline src/components/Exchange.js]: Isn't this still needed?

File: src/components/Header.js [hunk 13]
```diff
@@ -92,27 +92,23 @@ export  default ({openScanner, network, total, dollarDisplay, ens, title, titleI
 
   let topLeft
 
-  const explorerUrl = network === 'Leap Network'
```
Comment (TimDaub) [inline src/components/Header.js]: Ah nice!

File: src/components/Exchange.js (hunk 12 above)
Comment (sunify) [reply src/components/Exchange.js]: No, variable was unused

File: src/components/Balance.js (hunk 10 above)
Comment (sunify) [reply src/components/Balance.js]: Because variable was unused (there were only assignments, and no read)

File: src/i18n/index.js [hunk 14]
```diff
@@ -1,6 +1,6 @@
 import i18next from "i18next";
 import LanguageDetector from "i18next-browser-languagedetector";
-import { fr, en, es, ca, de, ro, he, ru, pt, ja } from "./locales";
```
Comment (TimDaub) [inline src/i18n/index.js]: lol, there is a french translation file. Why remove?

Comment (sunify) [reply src/i18n/index.js]: It's unused. I tried not change app's behavior in this PR

**Comment by sunify** [comment]
> This might also a good one to send upstream :)

Definitely 👌

File: src/i18n/index.js (hunk 14 above)
Comment (TimDaub) [reply src/i18n/index.js]: OK weird. Created an issue: https://github.com/leapdao/burner-wallet/issues/135

**Pull request merged**